src/components/TranslationPanel.js

```diff
@@ -11,11 +11,17 @@ const i18n = new(require('../translations/i18n'));
 const db = require(`${__dirname}/../util/data-provider`).targetDb();
 import Statistic  from '../components/Statistic';
 import { FormattedMessage } from 'react-intl';
+import { CommunicationSpeakerPhone } from 'material-ui/svg-icons';
 
+var el;
+var range;
+var sel;
+var pos;
+var textNode;
 @observer
 class TranslationPanel extends React.Component {
 	constructor(props){
-    	super(props);
```

Comment:
Let's revert all whitespace changes. In this case, it doesn't seem to add any value and looks off. Make sure your editor has the correct defaults same as that of @sandeeponrails.

Reply:
Changes has been made(tab space set to 4) on my vscode as per @sandeeponrails settings